### PR TITLE
Dashboard proposal

### LINKED ISSUE #271: Dashboard
The main page (home) should be the Unleash timeline consisting of a list of:
- announcements of someone achieving a goal
- custom prepared news

Prepare a implementation with mocked data for this.
## Synopsis

> "Since we added the Slack integration to the Unleash app - and all updates are being posted automagically, the Unleash Timeline has been somewhat forgotten (unleash-timeline.x-team.com). I'd like to make the Timeline a part of Unleash 2.0 and use it to not only present all the self explanatory goals developers achieve (and thus have them presented there in addition to the Slack channel), but also share all "bigger" news that would require a bit more context than just one sentence to give a full picture."

### PR DESCRIPTION
**Pull request by rmariuzzo**
This fix #271

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,8 +44,10 @@
     "lodash": "^4.15.0",
     "material-ui": "^0.15.2",
     "react": "^15.2.1",
+    "react-chronos": "file:///Users/rubens/Workspace/react-chronos",
```
Comment (kdemoya) [inline package.json]: Forgot to replace this one :P, looks like that's why tests are failing.

Comment (karolsojko) [inline package.json]: also @rmariuzzo make sure to update the yarn.lock file ;)

**Comment by karolsojko** [comment]
I guess the tests fail due to the `yarn.lock` being not updated with the changes

**Comment by nadeemkhedr** [comment]
Another issue I am getting 
```
Warning: Unknown props `eventSelector`, `markerSelector`, `eventClassNames`, `markerClassNames` on <div> tag. Remove these props from the element
```

The reason I think is the following line in [react-chronos](https://github.com/rmariuzzo/react-chronos/blob/master/src/Chronology.js#L182l) which passed all the properties from the component to the div , can you check that @rmariuzzo

**Pull request merged**